crates/test-utils/src/deploy_helpers.rs

```diff
@@ -831,6 +746,71 @@ impl KakarotTestEnvironmentContext {
         Self { sequencer, kakarot_client, kakarot, kakarot_contract, evm_contracts }
     }
 
+    /// Initializes the Kakarot test environment and deploys the Kakarot system and EVM contracts.
+    async fn initialize_without_dumped_state(
+        sequencer: TestSequencer,
+        starknet_provider: Arc<JsonRpcClient<HttpTransport>>,
+    ) -> Self {
+        let starknet_account = sequencer.account();
+
+        // Define the expected funded amount for the Kakarot system
```

Comment:
in which sense is this amount "expected" ?

Reply:
Should be `funding_amount`, will fix